packages/workbox-window/src/Workbox.ts

```diff
@@ -40,7 +42,25 @@ const REGISTRATION_TIMEOUT_DURATION = 60000;
  *
  * @memberof module:workbox-window
  */
-class Workbox extends EventTargetShim {
+class Workbox extends WorkboxEventTarget {
+  private _scriptURL: 'string';
+  private _registerOptions: RegistrationOptions = {};
+  private _updateFoundCount: number = 0;
+
+  // Deferreds we can resolve later.
+  private _swDeferred: Deferred<ServiceWorker> = new Deferred();
+  private _activeDeferred: Deferred<ServiceWorker> = new Deferred();
+  private _controllingDeferred: Deferred<ServiceWorker> = new Deferred();
+
+  private _registrationTime: DOMHighResTimeStamp = 0;
+  private _isUpdate?: Boolean;
```

Comment:
Similar question about `Boolean` vs. `boolean`, with the lower-case version being what I see at https://www.typescriptlang.org/docs/handbook/basic-types.html#boolean

Is there a linting rule that might catch this sort of thing, to ensure consistency if both are supported?

Reply:
Changed to `boolean`. As for linting, my plan was to add that to eslint at some point, but I haven't looked into it yet.